sycl/test-e2e/Basic/diagnostics/device-check.cpp

```diff
@@ -1,14 +1,14 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=cpu %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=gpu %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=acc %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=CPU %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=GPU %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=ACC %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=Cpu %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=Gpu %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=Acc %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='%sycl_be:*' SYCL_DEVICE_TYPE=XPU %t.out
+// RUN: env SYCL_DEVICE_TYPE=cpu %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=gpu %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=acc %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=CPU %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=GPU %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=ACC %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=Cpu %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=Gpu %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=Acc %BE_RUN_PLACEHOLDER t.out
+// RUN: env SYCL_DEVICE_TYPE=XPU %BE_RUN_PLACEHOLDER t.out
```

Comment:
Can't do that, unfortunately. `%*_RUN_PLACEHOLDER`s might contain `%RUN_LAUNCHER` and it has to come right before the executable. Also, other placeholder are currently used like this, so I suspect it might be the definition of this particular one that needs to be fixed.